client/src/app/app.module.css

```diff
@@ -0,0 +1,43 @@
+button {
```

Comment:
global styles should be in client/src/index.css not here.

Reply:
Done.